python/GafferUI/MenuBar.py

```diff
@@ -125,10 +127,11 @@ def __parentChanged( self, widget ) :
 # shortcuts in order to defer to our own code here.
 class _ShortcutEventFilter( QtCore.QObject ) :
```

Comment:
Wasn't sure if this should then become `ShortcutEventFilter`. I know we like to minimise the public API surface, but its less apparent now that this is in use elsewhere.